Api/Data/Models/Thread.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SharpScape.Api.Models;
+// cant use Thread as classname cause its already used in namespace. So I name it ThreadModel
+public class ThreadModel
+{
+    [Key]
+    public Guid Id{get;set;}
+    [ForeignKey("User")]
+    public Guid UserId {get;set;}
+    public User User{get;set;}
+    public string Title { get; set; }
+    public string Body {get;set;}
+    public string Votes { get; set; }
+    public string Relies { get; set; }
```

Comment:
```suggestion
    public int Replies { get; set; }
```
Fixed typo.  Also I think this ought to be a number -- this can be a made-up number for now; later, it can be a count of the number of `Post`s which refer to this thread's `Id`

Reply:
yes forget that one. I will just remove this attribute Replies. it will be postid (foreignkey from post table)

Reply:
I think the model & DTO actually should contain a `int Replies`, to make it easily display how many posts there are, without having to query the whole `Posts` table and count them

Reply:
Oh right.